packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/vsphere/validateVCenterURL.ts

```diff
@@ -1,6 +1,10 @@
+import { useForkliftTranslation } from 'src/utils';
+
 import { validateURL, ValidationMsg } from '../../common';
 
 export const validateVCenterURL = (url: string | number): ValidationMsg => {
+  const { t } = useForkliftTranslation();
```

Comment:
I don't want to have hook calls outside of components, `useForkliftTranslation` is a hook and `validateVCenterURL` is a utility method that can be called from a none component entity.

lets wait with this refactor, it's not connected to the vsphere thing, the i18n of validation msgs should have a better fix.